content/blog/coredns-in-kubernetes-via-kubeadm.md

```diff
@@ -10,11 +10,33 @@ We can now install CoreDNS as the default service discovery via Kubeadm, which i
 
 Currently, [CoreDNS is Alpha in Kubernetes 1.9](https://github.com/kubernetes/features/issues/427). We have a roadmap which will make CoreDNS Beta in version 1.10 and eventually be the default DNS, replacing kube-dns.
 
+>It is important to note that currently when switching from kube-dns to CoreDNS, the configurations that come with kube-dns (stubzones, federations...) will no longer exist and will switch to a default configuration in CoreDNS.
+>The translation of the configurations from kube-dns shall be supported from the upcoming version of Kubernetes (v1.10) where CoreDNS will be Beta.
+
+
+## [Plugins](https://coredns.io/plugins/)
+CoreDNS in Kubernetes ships with the following `plugins` enabled:
+- *[errors](https://coredns.io/plugins/errors/)*: This enables error logging.
+- *[health](https://coredns.io/plugins/health/)*: health enables a simple health check endpoint.
+- *[kubernetes](https://coredns.io/plugins/kubernetes/)*: The kubernetes plugin enables the reading zone data from a Kubernetes cluster. You can find more details [here](https://coredns.io/plugins/kubernetes/). 
+
+> The `pods insecure` option always return an A record with IP from request (without checking k8s). This option is provided for backward compatibility with kube-dns.
+> Also by default, the Kubernetes plugin has the `Cluster Domain` and the `Service CIDR` defined. The `Pod CIDR` must be added to the config file after CoreDNS deployment.
```

Comment:
Hum.. what is the process to update 'Pod CIDR' ? that is not done by the upgrade process ?